code/__DEFINES/antagonists.dm

```diff
@@ -87,3 +87,12 @@ GLOBAL_LIST_INIT(heretic_start_knowledge,list(/datum/eldritch_knowledge/spell/ba
 
 /// How much does it cost to reroll strains?
 #define BLOB_REROLL_COST 40
+
+/// How many telecrystals a normal traitor starts with
+#define TELECRYSTALS_DEFAULT 20
```

Comment:
Are we sure there isn't already a way to pull this value from the code somewhere without a new define? Seems very odd to make a new define like this but not actually apply it in the code where uplinks are given 20 TCs

Reply:
You are totally correct, I forgot to use it in the actual place where the 20 TC is defined.